srcpkgs/ardour/template

```diff
@@ -13,26 +13,21 @@ makedepends="boost-devel liblrdf-devel liblo-devel lilv-devel suil-devel
  vamp-plugin-sdk-devel fftw-devel jack-devel libsamplerate-devel lv2
  sratom-devel gtkmm2-devel libarchive-devel fluidsynth-devel hidapi-devel
  libltc-devel glibmm-devel"
+depends="desktop-file-utils"
 short_desc="Professional-grade digital audio workstation"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="http://ardour.org"
 distfiles="https://community.ardour.org/src/Ardour-${version}.tar.bz2"
-checksum=93418636d7d93f66bdf479ed5bf3c6b0cc0198dc8f76c5555110d3147e4ec74b
+checksum=ac981ec7565fa8a171116bc2fbeb3198a01aad4b7d7c846f965e074a8a162bdb
 
 case "$XBPS_TARGET_MACHINE" in
 	x86_64*)  configure_args+=" --dist-target=x86_64" ;;
 	i686*)    configure_args+=" --dist-target=i686" ;;
 	aarch64*) configure_args+=" --dist-target=aarch64" ;;
 	ppc64*)   configure_args+=" --dist-target=ppc64" ;;
 	ppc*)     configure_args+=" --dist-target=ppc" ;;
-	# Seriously, build system for arm is broken
-	# They don't use cflags from pkg-config
-	# We need to append output of
-	# pkg-config --cflags glibmm-2.4 libxml-2.0
-	# into both CFLAGS and CXXFLAGS, and it's still run into failure.
-	# Maybe better with NEON on, I don't know
-	arm*)     broken="You are compiling libardour on a platform for which ardour/cycles.h needs work" ;;
+	armv7*)   configure_args+=" --dist-target=armhf" ;;
```

Comment:
Did you try armv6l as well? Our target is pretty much armhf.

Reply:
I think they used armhf for Armv7 CPU with Thumb-2 and VFP3D16 like Debian does (https://wiki.debian.org/ArmHardFloatPort/) because it's used to enable NEON instructions, which ARMv6 and lower do not support.

Reply:
I'll try if it builds but can't test it...

Reply:
If it has a headless version I can test locally.

Reply:
seems to work